src/kouros/pos/pos_manager.go

```diff
@@ -161,4 +161,17 @@ type POSManager interface {
 	// Stop the collection of telemetry data in PoseidonOS
 	// The function takes a protobuf format as parameter and returns response in protobuf format
 	StopTelemetry() (*pb.StopTelemetryResponse, error)
+
+
+    // Create the volume in PoseidonOS Array
+    // The function takes a protobuf format as parameter and returns response in protobuf format
+    CreateVolume(param *pb.CreateVolumeRequest_Param) (*pb.CreateVolumeResponse, error)
```

Comment:
Looks like the code is not indendented properly. Please run go fmt to format it

Reply:
Thanks for pointing out, formatting is done